core/trino-server-rpm/src/test/java/io/trino/server/rpm/ServerIT.java

```diff
@@ -42,7 +42,7 @@
 @Test(singleThreaded = true)
 public class ServerIT
 {
-    private static final String BASE_IMAGE = "prestodev/centos7-oj11";
+    private static final String BASE_IMAGE = "ghcr.io/trinodb/testing/centos7-oj11";
```

Comment:
Does it work locally? Do I need to log in anywhere?

Reply:
They are public